commit 583a69ae2d20a57e6aeabeaf45d7b022ba43c276
Author: Daniel P. Berrange <berrange@redhat.com>
Date:   Thu Oct 3 12:35:34 2013 +0100

    Remove virConnectPtr arg from virNWFilterDefParse*
    
    None of the virNWFilterDefParse* methods require a virConnectPtr
    arg, so just drop it
    
    Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
    (cherry picked from commit b77b16ce4166dcc87963ae5d279b77b162ddbb55)

Index: libvirt-1.1.1/src/conf/nwfilter_conf.c
===================================================================
--- libvirt-1.1.1.orig/src/conf/nwfilter_conf.c	2014-03-25 14:15:25.098627506 -0500
+++ libvirt-1.1.1/src/conf/nwfilter_conf.c	2014-03-25 14:15:25.090627506 -0500
@@ -2633,8 +2633,7 @@ cleanup:
 
 
 static virNWFilterDefPtr
-virNWFilterDefParse(virConnectPtr conn ATTRIBUTE_UNUSED,
-                    const char *xmlStr,
+virNWFilterDefParse(const char *xmlStr,
                     const char *filename) {
     virNWFilterDefPtr def = NULL;
     xmlDocPtr xml;
@@ -2649,18 +2648,16 @@ virNWFilterDefParse(virConnectPtr conn A
 
 
 virNWFilterDefPtr
-virNWFilterDefParseString(virConnectPtr conn,
-                          const char *xmlStr)
+virNWFilterDefParseString(const char *xmlStr)
 {
-    return virNWFilterDefParse(conn, xmlStr, NULL);
+    return virNWFilterDefParse(xmlStr, NULL);
 }
 
 
 virNWFilterDefPtr
-virNWFilterDefParseFile(virConnectPtr conn,
-                        const char *filename)
+virNWFilterDefParseFile(const char *filename)
 {
-    return virNWFilterDefParse(conn, NULL, filename);
+    return virNWFilterDefParse(NULL, filename);
 }
 
 
@@ -3055,7 +3052,7 @@ virNWFilterObjLoad(virConnectPtr conn,
     virNWFilterDefPtr def;
     virNWFilterObjPtr nwfilter;
 
-    if (!(def = virNWFilterDefParseFile(conn, path))) {
+    if (!(def = virNWFilterDefParseFile(path))) {
         return NULL;
     }
 
Index: libvirt-1.1.1/src/conf/nwfilter_conf.h
===================================================================
--- libvirt-1.1.1.orig/src/conf/nwfilter_conf.h	2014-03-25 14:15:25.098627506 -0500
+++ libvirt-1.1.1/src/conf/nwfilter_conf.h	2014-03-25 14:15:25.090627506 -0500
@@ -713,10 +713,8 @@ int virNWFilterLoadAllConfigs(virConnect
 char *virNWFilterConfigFile(const char *dir,
                             const char *name);
 
-virNWFilterDefPtr virNWFilterDefParseString(virConnectPtr conn,
-                                            const char *xml);
-virNWFilterDefPtr virNWFilterDefParseFile(virConnectPtr conn,
-                                          const char *filename);
+virNWFilterDefPtr virNWFilterDefParseString(const char *xml);
+virNWFilterDefPtr virNWFilterDefParseFile(const char *filename);
 
 void virNWFilterObjLock(virNWFilterObjPtr obj);
 void virNWFilterObjUnlock(virNWFilterObjPtr obj);
Index: libvirt-1.1.1/src/nwfilter/nwfilter_driver.c
===================================================================
--- libvirt-1.1.1.orig/src/nwfilter/nwfilter_driver.c	2014-03-25 14:15:25.098627506 -0500
+++ libvirt-1.1.1/src/nwfilter/nwfilter_driver.c	2014-03-25 14:15:25.090627506 -0500
@@ -561,7 +561,7 @@ nwfilterDefineXML(virConnectPtr conn,
     nwfilterDriverLock(driver);
     virNWFilterCallbackDriversLock();
 
-    if (!(def = virNWFilterDefParseString(conn, xml)))
+    if (!(def = virNWFilterDefParseString(xml)))
         goto cleanup;
 
     if (virNWFilterDefineXMLEnsureACL(conn, def) < 0)
Index: libvirt-1.1.1/tests/nwfilterxml2xmltest.c
===================================================================
--- libvirt-1.1.1.orig/tests/nwfilterxml2xmltest.c	2014-03-25 14:15:25.098627506 -0500
+++ libvirt-1.1.1/tests/nwfilterxml2xmltest.c	2014-03-25 14:15:25.090627506 -0500
@@ -36,7 +36,7 @@ testCompareXMLToXMLFiles(const char *inx
 
     virResetLastError();
 
-    if (!(dev = virNWFilterDefParseString(NULL, inXmlData))) {
+    if (!(dev = virNWFilterDefParseString(inXmlData))) {
         if (expect_error) {
             virResetLastError();
             goto done;
